### PR TITLE
[New Rule] Suspicious PowerShell Engine ImageLoad

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/558

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
image

File: rules/windows/defense_evasion_suspicious_powershell_imgload.toml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+license = "Elastic License"
+name = "Suspicious PowerShell Engine ImageLoad"
+risk_score = 43
+rule_id = "852c1f19-68e8-43a6-9dce-340771fe1be3"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+library where file.name == "System.Management.Automation.ni.dll" and
```
Comment (dstepanic17) [inline rules/windows/defense_evasion_suspicious_powershell_imgload.toml:25]: Can we add the other related PowerShell engine DLL (System.Management.Automation.dll) too?  I know when I did previous testing with SharpPick, we record the image load, but it wasn't part of the rule. 

image


```suggestion
library where file.name in ("System.Management.Automation.ni.dll", "System.Management.Automation.dll") and
```

Comment (Samirbous) [reply rules/windows/defense_evasion_suspicious_powershell_imgload.toml:25]: good catch

**Review by dstepanic17 — APPROVED** [review]
LGTM

File: rules/windows/defense_evasion_suspicious_powershell_imgload.toml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+description = """
+Identifies the PowerShell engine being invoked by unexpected processes. Rather than executing PowerShell functionality
+with powershell.exe, some attackers do this to operate more stealthily.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious PowerShell Engine ImageLoad"
+risk_score = 43
```
Comment (bm11100) [inline rules/windows/defense_evasion_suspicious_powershell_imgload.toml:17]:
```suggestion
risk_score = 47
```

File: rules/windows/defense_evasion_suspicious_powershell_imgload.toml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+rule_id = "852c1f19-68e8-43a6-9dce-340771fe1be3"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+library where file.name : ("System.Management.Automation.ni.dll", "System.Management.Automation.dll") and
+/* add False Positives relevant to your environment here */
```
Comment (bm11100) [inline rules/windows/defense_evasion_suspicious_powershell_imgload.toml:25]:
```suggestion
/* add false positives relevant to your environment here */
```

**Review by bm11100 — APPROVED** [review]

**Pull request merged**